source/grid/grid_generator.cc

```diff
@@ -28,11 +29,1186 @@
 #include <cmath>
 #include <limits>
 
+
 DEAL_II_NAMESPACE_OPEN
 
 
 namespace GridGenerator
 {
+  namespace Airfoil
+  {
+    AdditionalData::AdditionalData()
+      // airfoil configuration
+      : airfoil_type("NACA")
+      , naca_id("2412")
+      , joukowski_center(-0.1, 0.14)
+      , airfoil_length(1.0)
+      // far field
+      , height(30.0)
+      , length_b2(15.0)
+      // mesh
+      , incline_factor(0.35)
+      , bias_factor(2.5)
+      , refinements(2)
+      , n_subdivision_x_0(3)
+      , n_subdivision_x_1(2)
+      , n_subdivision_x_2(5)
+      , n_subdivision_y(3)
+      , airfoil_sampling_factor(2)
+    {
+      Assert(
+        airfoil_length <= height,
+        ExcMessage(
+          "Mesh is to small to enclose airfoil! Choose larger field or smaller"
+          " chord length!"));
+      Assert(incline_factor < 1.0 && incline_factor >= 0.0,
+             ExcMessage("incline_factor has to be in [0,1)!"));
+    }
+
+
+
+    void
+    AdditionalData::add_parameters(ParameterHandler &prm)
+    {
+      prm.enter_subsection("FarField");
+      {
+        prm.add_parameter("Height", height, "CoarseGridBlockHight");
+        prm.add_parameter("LengthB2", length_b2, "LengthBlock2");
+        prm.add_parameter("InclineFactor",
+                          incline_factor,
+                          "Define obliqueness at boarder block1 to block2");
+      }
+      prm.leave_subsection();
+
+      prm.enter_subsection("AirfoilType");
+      {
+        prm.add_parameter("Type",
+                          airfoil_type,
+                          "Type of airfoil geometry",
+                          Patterns::Selection("NACA|Joukowski"));
+      }
+      prm.leave_subsection();
+
+      prm.enter_subsection("NACA");
+      {
+        prm.add_parameter("NacaId", naca_id, "Naca ID");
+      }
+      prm.leave_subsection();
+
+      prm.enter_subsection("Joukowski");
+      {
+        prm.add_parameter("Center",
+                          center,
```

Comment:
```suggestion
                          joukowski_center,
```

Reply:
@masterleinad Thanks! I have changed this!